armadas/scavenger_lair_27.json

```diff
@@ -11,7 +11,7 @@
   "rarity": "uncommon",
   "rewards": [
     {
-      "type": "1",
+      "type": "1\u21f4 Raw Isogen",
```

Comment:
Are we not using rarity and grade here?

Reply:
If we use grade, then the code will assume it's Ore, Crystal or Gas. This falls under `other` for the code.

Reply:
OK, I will need to adjust my parser for new content to that as well :+1: